docs/appstate.md

```diff
@@ -157,20 +157,21 @@ Received when the user is not actively interacting with the app. Useful in situa
 ### `addEventListener()`
 
 ```jsx
-addEventListener(type, handler);
+addEventListener(eventType, listener);
 ```
 
-Add a handler to AppState changes by listening to the `change` event type and providing the handler
+Sets up a function that will be called whenever the specified event type on AppState occurs. Valid values for `eventType` are
+[listed above](https://reactnative.dev/docs/appstate#events).  Returns the `EventSubscription`.
```

Comment:
```suggestion
[listed above](#events). Returns the `EventSubscription`.
```

This suggestion should fix the Lint error. It also suggest the switch from the absolute link to the relative one.